snuba/utils/constants.py

```diff
@@ -5,3 +5,6 @@
 
 #: Metrics granularities for which a materialized view exist, in ascending order
 GRANULARITIES_AVAILABLE = (10, 60, 60 * 60, 24 * 60 * 60)
+
+# Number of EAP Span buckets
```

Comment:
should probably have something here like explaining what it would mean to change this constant

Reply:
agree, also change the other places that use this constant to also use this one - see my PR for example https://github.com/getsentry/snuba/pull/6173/files